config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -346,25 +346,24 @@ presubmits:
             - --
             - --cluster=
             - --extract=ci/latest
-            - --gcp-master-size=n1-standard-4
-            - --gcp-node-size=n1-standard-8
-            - --gcp-nodes=7
+            - --gcp-master-size=n1-standard-2
+            - --gcp-node-size=n1-standard-4
+            - --gcp-nodes=4
             - --gcp-project=k8s-presubmit-scale
             - --gcp-zone=us-east1-b
             - --kubemark
-            - --kubemark-nodes=500
+            - --kubemark-nodes=100
             - --provider=gce
             - --tear-down-previous
             - --test=false
             - --test_args=--ginkgo.focus=xxxx
             - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
             - --test-cmd-args=cluster-loader2
-            - --test-cmd-args=--nodes=500
+            - --test-cmd-args=--nodes=100
             - --test-cmd-args=--provider=kubemark
             - --test-cmd-args=--report-dir=/workspace/_artifacts
             - --test-cmd-args=--testconfig=testing/density/config.yaml
             - --test-cmd-args=--testconfig=testing/load/config.yaml
-            - --test-cmd-args=--testoverrides=./testing/load/kubemark/500_nodes/override.yaml
```

Comment:
```
--test-cmd-args=--testoverrides=./testing/load/kubemark/throughput_override.yaml
```
It will make test slightly faster + coherent with kubemark-100 ci-job.

Reply:
Done.